spec/features/user_creates_list_spec.rb

```diff
@@ -24,10 +24,21 @@
 
     expect(page).to have_content("List saved")
 
-    @voters.each do |voter|
-      expect(page).to have_content(voter.full_name)
+    @voters[0..24].each do |voter|
+      expect(page).to have_content(voter.first_name)
+    end
+
+    click_link 'Next'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.